docs/ja/advanced-config/README.md

```diff
@@ -1,16 +1,14 @@
 # 高度な設定
 
-Starship は汎用性の高いシェルですが、時には特定の処理を行うために `starship.toml` を編集する以上のことをする必要があります。このページでは starship で使用される、より高度な設定の一部を詳しく説明していきます。
+Starship は汎用性の高いシェルですが、時には特定の処理を行うために `starship.toml` を編集する以上のことをする必要があります。 このページでは starship で使用される、より高度な設定の一部を詳しく説明していきます。
 
-::: warning
-ここに載せられた設定は、Starship の将来のリリースで変更される可能性があります。
- :::
+::: warning ここに載せられた設定は、Starship の将来のリリースで変更される可能性があります。 :::
```

Comment:
It looks like these blocks are breaking because the `:::` on either side are not on new lines. Looking to fix these before we can merge.